prime-react/src/main/scala/react/primereact/AccordionMultiple.scala

```diff
@@ -22,10 +22,14 @@ case class AccordionMultiple(
   onTabOpen:     js.UndefOr[Int => Callback] = js.undefined,
   onTabClose:    js.UndefOr[Int => Callback] = js.undefined,
   onTabChange:   js.UndefOr[Int => Callback] = js.undefined,
-  tabs:          List[AccordionTab] = List.empty
+  tabs:          List[AccordionTab] = List.empty,
+  modifiers:     Seq[TagMod] = Seq.empty
 ) extends ReactFnProps[AccordionBase](AccordionBase.component)
     with AccordionBase {
 
+  def addModifiers(modifiers: Seq[TagMod]) = copy(modifiers = this.modifiers ++ modifiers)
```

Comment:
Could these be part of `ReactFnProps`? or another common trait?

Reply:
I think that the trait would have to know how to `copy`. Or use a mutable accumulator.

BTW, could we have a `def apply(mods: TagMod*)` method?

Reply:
> I think that the trait would have to know how to `copy`. Or use a mutable accumulator.

Right. I don't see a way around this. So, I don't think there is a lot of benefit to having a trait.